Mathlib/Topology/ContinuousFunction/Bounded.lean

```diff
@@ -1626,4 +1626,26 @@ lemma norm_sub_nonneg (f : α →ᵇ ℝ) :
 
 end
 
+variable {𝕜 : Type*} [NormedField 𝕜] [TopologicalSpace α] [NormedRing γ]
+
+/-- The subtype of compactly supported functions as an ideal. -/
+def CompactlySupportedBoundedContinuousFunction : Ideal (α →ᵇ γ) where
+  carrier := { f : α →ᵇ γ | HasCompactSupport f }
+  add_mem' := by
+    intro _ _ h₁ h₂
+    exact h₁.add h₂
+  zero_mem' := by simp [HasCompactSupport, tsupport]
+  smul_mem' := by
+    intro _ _ h
+    rw [mem_setOf_eq, smul_eq_mul, coe_mul]
+    exact h.mul_left
+
+@[inherit_doc]
+scoped[BoundedContinuousFunction] notation (priority := 2000)
+  "C_cb(" α ", " γ ")" => CompactlySupportedBoundedContinuousFunction α γ
+
+@[inherit_doc]
+scoped[BoundedContinuousFunction] notation α " →C_cb " γ =>
+  CompactlySupportedBoundedContinuousFunction α γ
```

Comment:
I know you did this because you saw it elsewhere, but I don't think we need both notation. Can you just remove this one and remove the priority assignment on the other?

Reply:
I'm not sure if I understood correctly: you would like that `CompactlySupportedBoundedContinuousFunction` to be `RingCon`, not `Ideal`, right? Is this notation still OK?